rules/windows/process_creation/proc_creation_win_sqlite_chromium_profile_data.yml

```diff
@@ -0,0 +1,41 @@
+title: SQLite Chromium Profile Data DB Access
+id: 24c77512-782b-448a-8950-eddb0785fc71
+status: experimental
+description: Detect use of sqlite binary to query databases in Chromium-based browsers for potentially sensitive user profile data, and steal the data contained within them
```

Comment:
```suggestion
description: Detect usage of the "sqlite" binary to query databases in Chromium-based browsers for potential data stealing.
```

Reply:
Thanks again for the support and reviews on these!